float32 IR0
float32 IR1
float32 IR2
float32 IR3
float32 IR4
float32 BUMP0
float32 BUMP1
